internal/pkg/deploy/cloudformation/stack/app.go

```diff
@@ -54,13 +54,16 @@ const (
 	appAdminRoleParamName         = "AdminRoleName"
 	appExecutionRoleParamName     = "ExecutionRoleName"
 	appDNSDelegationRoleParamName = "DNSDelegationRoleName"
-	appOutputKMSKey               = "KMSKeyARN"
-	appOutputS3Bucket             = "PipelineBucket"
-	appOutputECRRepoPrefix        = "ECRRepo"
-	appDNSDelegatedAccountsKey    = "AppDNSDelegatedAccounts"
-	appDomainNameKey              = "AppDomainName"
-	appDomainHostedZoneIDKey      = "AppDomainHostedZoneID"
-	appNameKey                    = "AppName"
+	// Linked to AppRegionalResources.KMSKeyARN
```

Comment:
We could put the comments on the same lines as the const declarations; might be a bit tidier. Up to you!